### ISSUE TITLE
[Accessibility][ScreenReader]: NVDA/Narrator are not reading the ToolTip message

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#2073]
* .NET Core Version: 3.1.0-preview1.19458.7
* Have you experienced this same bug with .NET Framework?: Yes

**Original bug:**
- [Bug 987952](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/987952)

**Problem description:**
- A user can't hear a tooltip text using NVDA/Narrator

**Actual behavior:** 
- NVDA/Narrator don't read a message from shown ToolTip

**Expected behavior:**
- NVDA/Narrator should announce ToolTip text
![image](https://user-images.githubusercontent.com/49272759/66656755-453d7700-ec47-11e9-9ed1-9cb8c857e030.png)

**Minimal repro:**
- Create .Net Core app
- Add some control and ToolTip
- Connect the control with the ToolTip
- Run the test app and NVDA/Narrator
- Listen what NVDA/Narrator say

**Test app:**
- [[Core][NVDA]ToolTipError.zip](https://github.com/dotnet/winforms/files/3717915/Core.NVDA.ToolTipError.zip)

**Comment by Lydia-Shi** [comment]
Verified the issue with .NET Core 3.1 Preview 3 -014638 build, the issue has been fixed, narrator can read tooltip message. Please see the below screenshot:
![read](https://user-images.githubusercontent.com/54019172/68186070-27f98f80-ffde-11e9-8642-2abc4f1bad47.PNG)
